lib/chat_web/live/main_live/index.html.heex

```diff
@@ -27,6 +27,7 @@
             let={f}
             for={:login}
             id="login-form"
+            class="t-form"
```

Comment:
we need a better naming 
`t-form` is too general

This should be something specific